issue99_test.go

```diff
@@ -0,0 +1,78 @@
+package zmq4
```

Comment:
please add the needed License blurb (with 2020)

also, I'd rename the file as `rep_test.go`.
or I'd add that test inside `zmq4_reqrep_test.go`

Reply:
Hi!
Will move the test into zmq4_reqrep_test.go (TestIssue99).